src/sentry/api/bases/dashboard.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import
```

Comment:
I modeled this a bit off of discover, but does anyone know if `sentry/api/bases` is the desired place to put this? It feels odd to me to put it here, but I didn't see a better alternative.

Reply:
Bases is probably not the right place. Where is this serializer used? I don't see it in this diff.

Reply:
Nm,  looks like you removed this while I was reviewing